kube-app-testing.sh

```diff
@@ -28,7 +28,7 @@ PYTHON_TESTS_DIR="test/kat"
 
 # gs cluster config
 DEFAULT_PROVIDER="aws"
-DEFAULT_GS_API_URL="https://api.g8s.gauss.eu-central-1.aws.gigantic.io"
+DEFAULT_GS_API_URL="https://api.g8s.gorilla.eu-central-1.aws.gigantic.io"
```

Comment:
Curious what was rationale for this switch?

Also, print_help still mentions gauss as default

Reply:
good catch, thanks Stevo. We switched to Gorilla because we kept getting run failures when using test installations - we were hitting issues with people testing components. These runs are quite expensive time-wise (because of how long it takes for the TC to come up), and we were finding ourselves mostly having to troubleshoot installation issues rather than actual test failures.

Reply:
Thanks, makes sense. Matches my experience with using test CPs. Even before testing begins have to first spend time cleaning up.